pyhf/interpolators/__init__.py

```diff
@@ -20,6 +20,7 @@ def _slow_interpolator_looper(histogramssets, alphasets, func):
 from .code1 import code1, _slow_code1
 from .code2 import code2, _slow_code2
 from .code4 import code4, _slow_code4
+from .codex import codex, _slow_codex
```

Comment:
@lukasheinrich This all looks good, but what are your thoughts on @kratsg's proposed naming scheme?

Reply:
i don't like it since it messes up the alignment :-p .. j/k no strong optinion either way

Reply:
It isn't my most favorite to be honest, but I don't have a better name and it probably is better than `x`.